### DISCUSSION TITLE
Do we use an assets directory?

### DISCUSSION THREAD

**Discussion by joseph-mueller** [IviFoundation/ivifoundation.github.io#35] [General]
It seems to be common practice to keep images and other assets in an assets directory.  Should we move our images under an assets/images directory?

Note that I will be deleting almost everything in images as part of clean-up.

**Comment by alejandro5042** [comment]
Yes, according to the [official docs for Jekyll](https://jekyllrb.com/docs/step-by-step/07-assets/):

![image](https://user-images.githubusercontent.com/3143819/221376239-438ee3d2-bd28-49d0-8498-41d1ee060afb.png)

The [Ruby on Rails website](https://rubyonrails.org/) (a very popular one) also does this: https://github.com/rails/website/tree/main/assets

While not strictly necessary for a functional website, it would be nice to follow this convention. Go for it! 😄

**Comment by joseph-mueller** [comment]
Done with yesterdays PR's

**Comment by alejandro5042** [comment]
Thanks @joseph-mueller! Closing this discussion thread as finished.
